docs/sql-programming-guide.md

```diff
@@ -1607,13 +1600,13 @@ a regular multi-line JSON file will most often fail.
 
 {% highlight r %}
 # sc is an existing SparkContext.
-sqlContext <- sparkRSQL.init(sc)
+spark <- sparkRSQL.init(sc)
```

Comment:
R API is still in experimental status, and we haven't introduced `SparkSession` to SparkR yet.